system-validation-test/features/ingest_basic.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Basic Ingest API calls
+  Ingest collections for various formats
+
+  Background:
+    Given I use the authorization token from environment variable "CMR_TOKEN"
+    And the provider "PROV1" exists
```

Comment:
So is this limited to only local tets?

Reply:
It can be run in different environments, but the requirements for running are:
- CMR_TOKEN and CMR_ROOT environment variables are configured for the proper environment (localhost, SIT, etc)
- A provider with name of PROV1 exists

If these are not both satisfied, instead of failing the test will rather be shown as pending and skipped.

Reply:
Right, but does PROV1 exist?  And do we have an OK from ops to use that provider name if it doesn't?

Reply:
That's a good question. Who would I reach out to regarding that?

Reply:
It might be worth making that configurable.